doc/topics/the-event-loop-timers-and-nexttick.md

```diff
@@ -69,7 +69,7 @@ actually uses - are those above._
 
 * **timers**: this phase executes callbacks scheduled by `setTimeout()`
  and `setInterval()`.
-* **I/O callbacks**: most types of callback except timers, `setImmedate()`, close
+* **I/O callbacks**: most types of callback except timers, `setImmediate()`, close
```

Comment:
I *think* this is supposed to read as: all callbacks, but not the ones listed above under **timers**, the `setImmediate()` ones, and those listed under **close callbacks** which are run later…? But ack, it isn’t very clear…